### PR TITLE
Ensuring that the time gets updated

### PR DESCRIPTION
**Pull request by mortenbirkelund**

### CODE REVIEW

**Comment by mortenbirkelund** [comment]
Fixes #127

**Comment by mortenbirkelund** [comment]
@marco-streng any comments?

File: components/widgets/datetime/index.js [hunk 1]
```diff
@@ -24,7 +24,12 @@ export default class DateTime extends Component {
...
   componentDidMount () {
     const { interval } = this.props
-    this.timeout = setTimeout(() => this.setState({ date: new Date() }), interval)
+    this.timeout = setTimeout(() => {
+      this.setState({
+        date: new Date()
+      })
+      this.componentDidMount()
+    }, interval)
   }
```
Comment (danielbayerlein) [inline components/widgets/datetime/index.js]: I have created a small example, please have a look at it: https://codesandbox.io/s/q4l2q6yr76

Comment (mortenbirkelund) [reply components/widgets/datetime/index.js]: The reason why I wanted to stick to settimeout was to keep it aligned with the other modules. If you prefer the setinterval approach, then it is fine by me.

Comment (danielbayerlein) [reply components/widgets/datetime/index.js]: For the other components, the update handling is a bit different than here. We can use the example from the comment above as a reference.

Comment (mortenbirkelund) [reply components/widgets/datetime/index.js]: Changed according to the example

**Comment by mortenbirkelund** [comment]
@danielbayerlein done

**Pull request merged**